doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -687,6 +687,10 @@ Modem libraries
       Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
       In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
 
+  * Added:
+
+    * A new :c:enum:`LTE_LC_EVT_RAI_UPDATE` event and subscribing to the event.
```

Comment:
"subscribing to the event" is internal LTE LC functionality, needed to be able to send the new event. In my opinion it's unnecessary to mention it in the changelog.

Reply:
I added so that users know that it's taken care of and it's not completely internal because user will select whether to subscribe with `CONFIG_LTE_RAI_REQ `. Maybe that existing option could be mentioned here too.

Reply:
Changed.